modules/gapi/src/backends/fluid/gfluidcore_func.dispatch.cpp

```diff
@@ -207,6 +207,18 @@ ABSDIFFC_SIMD(float)
 
 #undef ABSDIFFC_SIMD
 
+#define SPLIT3_SIMD(SRC, DST)                                            \
+int split3_simd(const SRC in[], DST out1[], DST out2[],                  \
+                DST out3[], const int width)                             \
```

Comment:
It makes no sense to make this function a macro for this kernel. Make this function a normal function.